modules/test/playwright/tests/client-extension-web/main/pages/ClientExtensionsPage.ts

```diff
@@ -116,6 +114,10 @@ export class ClientExtensionsPage extends POM {
 
 		const erc = dataId.replace('string,', '').replace(':name', '');
 
+		// Now go to the view client extension page
+
+		await this.viewMenuItem.click();
+
```

Comment:
I wonder how cliking, then getting the erc could even work once

Reply:
If it goes fast enough there's time to obtain the ERC.